chrome_manifest_v3/class_app.js

```
@@ -0,0 +1,869 @@
+/** Gmail2Trello Application - ES6 Class Version
+ */
+
+var Gmail2Trello = Gmail2Trello || {}; // Namespace initialization - must be var to guarantee correct scope
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)